packages/components/empty-state/src/EmptyState.tsx

```diff
@@ -79,7 +79,7 @@ const _EmptyState = (props: EmptyStateProps, ref) => {
   } = props;
 
   return (
-    <Primitive
+    <Box
```

Comment:
Should this use `useBox` as well to avoid rendering `Box` in runtime?

Reply:
I'm not sure if we need to optimize to this level here. I think components should be able to use <Box> when necessary without too much penalty.